src/services/Content/ContentApi.ts

```diff
@@ -330,12 +330,30 @@ export const introspectRepository: (
   return data;
 };
 
+export const triggerSnapshot: (
+  repositoryUUID: string,
+  resetCount: number,
+) => Promise<void> = async (repositoryUUID) => {
+  try {
+    const { data } = await axios.post('/api/content_sources/v1.0/repositories/$UUID/snapshot/', {
+      repository_uuid: repositoryUUID,
+    });
```

Comment:
This won't work as is, as $UUID doesn't get replaced with the repository_uuid.  As i mentioned before look at https://github.com/content-services/content-sources-frontend/blob/main/src/services/Content/ContentApi.ts#L323-L331  for an example of how to replace the UUID properly.